client/jetpack-cloud/sections/partner-portal/lib/get-product-short-title.ts

```diff
@@ -0,0 +1,21 @@
+import getProductTitle from './get-product-title';
+import getProductVariantShortTitle from './get-product-variant-short-title';
+
+/**
+ * Get the short version of the product title
+ * @param product Product
+ * @param removeVariant if we need to remove variant information
+ * @returns Product title
+ */
+export default function getProductShortTitle(
+	product: string,
+	removeVariant: boolean = false
+): string {
+	const title = getProductTitle( product, removeVariant );
+
+	if ( title.startsWith( 'VaultPress Backup Add-on' ) ) {
```

Comment:
I'm curious if it is better to check `product.slug === "jetpack-backup-storage"` instead. WDYT?

Reply:
Yes, I agree checking the family slug is a more reliable approach. 👍 I made the changes.

We could also apply it to `getProductTitle`, but it will require more changes unrelated to this PR. I'm skipping `getProductTitle` for now to maintain PR size.